regulations/static/regulations/css/less/module/comment.less

```diff
@@ -4,12 +4,20 @@ comment.less
 comment.less contains styles related to commenting on sections
 */
 
+@comment_blue: #4A90E2;
+@comment_blue_dark: #00629f;
+@comment_blue_light: #BDDCFD;
+@comment_item_blue: #E2F3F8;
+@comment_item_gray: #AEB0B5;
+@comment_input_border: #E5E5E5;
```

Comment:
Yeah at the moment don't know if the colors are final or not, will def later replace variables if they are repeated by the ones already defined in `variables.less` and make better names for them once things are more finalized.